src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -141,7 +141,10 @@ public class ExtensionsRunner {
     private final ExtensionsIndicesModuleRequestHandler extensionsIndicesModuleRequestHandler = new ExtensionsIndicesModuleRequestHandler();
     private final ExtensionsIndicesModuleNameRequestHandler extensionsIndicesModuleNameRequestHandler =
         new ExtensionsIndicesModuleNameRequestHandler();
-    private final ExtensionsRestRequestHandler extensionsRestRequestHandler = new ExtensionsRestRequestHandler(extensionRestPathRegistry);
+    private final ExtensionsRestRequestHandler extensionsRestRequestHandler = new ExtensionsRestRequestHandler(
+        extensionRestPathRegistry,
+        sdkNamedXContentRegistry
```

Comment:
This isn't initialized yet so it would be null when you passed it here.   The extensionRestPathRegistry is initialized on  line 103 so it's fine.  So just declare the request handler variable here, and move the initialization down somewhere below where sdkNamedXContent is initialized (line 183)